Stylesheets/Components/badge.css

```diff
@@ -0,0 +1,119 @@
+.avatar-badge-container {
+    width:fit-content;
+    height: fit-content;
+    position: relative;
+}
+
+.avatar-badge-container .avatar-container, 
+.avatar-badge-container img {
+    margin:0;
+}
+
+.badge {
+    border-radius: var(--circle-border);
+    position:absolute;
+    right:0;
+    bottom:0;
+    z-index: 9;
+    box-shadow: var(--badge-shadow)
+}
+
+.squircle + .badge {
+    right: -8px;
+    bottom: -8px;
+}
+
+.squircle.xsmall + .badge {
+    right: -3px;
+    bottom: -3px;
+}
+
+.squircle.large + .badge {
+    right: -4px;
+    bottom: -4px;
+}
+
+.xsmall + .badge {
+    width: 1rem;
+    height: 1rem;
+    border: 3px solid var(--color-white);
+} 
+
+.small + .badge {
+    width: 1.5rem;
+    height: 1.5rem;
+    border: 4px solid var(--color-white);
+} 
+
+.large + .badge {
+    width: 2rem;
+    height: 2rem;
+    border: 5px solid var(--color-white);
+} 
+
+.xlarge + .badge {
+    width: 3rem;
+    height: 3rem;
+    border: 6px solid var(--color-white);
+} 
+
+.badge.online {
+    background-color:var(--color-success-solid);
+}
+
+.badge.offline {
+    background-color: var(--color-offline);
+}
+
+.badge.dnd {
+    background-color: var(--color-error-solid);
+}
+
+.badge.notification {
+    top:0;
+    border: none;
+    width : 1.5rem;
```

Comment:
the number inside the badge is not readable. you can increase the width and then use flex to center it or give some padding.

Reply:
okay, working on it. Thankyou!